components/Header/MobileFooter.vue

```diff
@@ -4,7 +4,7 @@
       <b-icon
```

Comment:
I made some edits to the plus button for create to match this standard better. The drop shadow matches material, and I removed the border, and inverted the colors to make the background green, and the plus sign white